src/new_hope/week1/day4.clj

```diff
@@ -0,0 +1,61 @@
+(ns new-hope.week1.day4
+  (:require [midje.sweet :refer :all]
+            [criterium.core :as crit]))
+
+;Penultimate	Element
+(def penultimate
+  #(last (take 2 (reverse %))))
```

Comment:
Besides `(last (take 2 ...))` there are also `(second ...)` and  `(fnext ...)`
And very inefficient `(last (butlast ...))`.